lib/galaxy/tool_util/deps/mulled/mulled_search.py

```diff
@@ -120,6 +120,8 @@ def get_json(self, search_string):
         Function takes search_string variable and returns results from the bioconda channel in JSON format
 
         """
+        if run_command is None:
+            raise Exception("Search with destination conda can only be run from a Conda environment.")
```

Comment:
You can also install `galaxy-tool-util[mulled]` (https://github.com/galaxyproject/galaxy/pull/10025)

Reply:
Oh awesome! I didn't realize pip install conda would work from outside conda - they had pushed back on that years ago. This is cool!

Reply:
It's just a small part of the API unfortunately, but it's enough for search to work

Reply:
Revised the error messages to make this usage more clear.